changelog/WRN2apJpQVqjcU9cRkanBw.md

```diff
@@ -0,0 +1,4 @@
+audience: users
+level: patch
+---
+Fix docker worker not working in the latest release of Taskcluster
```

Comment:
Since users see the exception, it might be good to include some keywords here, too, especially `taskVolumeBindings`.